packages/eslint-plugin-next/src/index.ts

```diff
@@ -25,39 +56,33 @@ module.exports = {
   configs: {
     recommended: {
       plugins: ['@next/next'],
-      rules: {
-        // warnings
-        '@next/next/google-font-display': 'warn',
-        '@next/next/google-font-preconnect': 'warn',
-        '@next/next/next-script-for-ga': 'warn',
-        '@next/next/no-async-client-component': 'warn',
-        '@next/next/no-before-interactive-script-outside-document': 'warn',
-        '@next/next/no-css-tags': 'warn',
-        '@next/next/no-head-element': 'warn',
-        '@next/next/no-html-link-for-pages': 'warn',
-        '@next/next/no-img-element': 'warn',
-        '@next/next/no-page-custom-font': 'warn',
-        '@next/next/no-styled-jsx-in-document': 'warn',
-        '@next/next/no-sync-scripts': 'warn',
-        '@next/next/no-title-in-document-head': 'warn',
-        '@next/next/no-typos': 'warn',
-        '@next/next/no-unwanted-polyfillio': 'warn',
-        // errors
-        '@next/next/inline-script-id': 'error',
-        '@next/next/no-assign-module-variable': 'error',
-        '@next/next/no-document-import-in-page': 'error',
-        '@next/next/no-duplicate-head': 'error',
-        '@next/next/no-head-import-in-document': 'error',
-        '@next/next/no-script-component-in-head': 'error',
-      },
+      rules: recommendedRules,
     },
     'core-web-vitals': {
       plugins: ['@next/next'],
       extends: ['plugin:@next/next/recommended'],
-      rules: {
-        '@next/next/no-html-link-for-pages': 'error',
-        '@next/next/no-sync-scripts': 'error',
-      },
+      rules: coreWebVitalsRules,
+    },
+  },
+}
+
+module.exports = plugin
+module.exports.flatConfig = {
+  recommended: {
+    name: 'eslint-plugin-next recommended',
```

Comment:
It's probably better to call it `next/recommended`, like other plugins do?

- https://github.com/un-ts/eslint-plugin-import-x/blob/ac6d2e1b39e2a0d01b981b98b58aad0eb7939c70/src/index.ts#L161
- https://github.com/Rel1cx/eslint-react/blob/c7c18c6af30d6a26108a7d47aaa0da6bd56bd404/packages/plugins/eslint-plugin/src/configs/recommended.ts#L10
- https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/7566e13531f09a040daee4c16d0cba0c28d321c4/src/index.js#L303
- https://github.com/sindresorhus/eslint-plugin-unicorn/blob/1774135a5ddbded2c89f82952e37a3e3bb01cdfa/index.js#L73

Same for `core-web-vitals` config.

Reply:
Done in [`8a7d85c` (#73873)](https://github.com/vercel/next.js/pull/73873/commits/8a7d85c1e791e6a0396d4592889c41c2e281ea59).